sandbox/data/cohort.json

```diff
@@ -0,0 +1,681 @@
+                "id": "9acd789d-c3be-4ca0-be63-e075aad0100f",
+                "logicallyAndRules": false,
+                "rules": [
+                  {
+                    "id": "bed9a06c-1d98-4629-a549-66063e4ebec5",
+                    "codeListInclusions": [
+                      {
+                        "id": "e5d7bc31-bed2-42a9-ad2f-b8f20b2f49e5",
+                        "shortDescription": "Chronic heart disease",
+                        "clusterName": "CHRONHD_COD",
+                        "codes": [
+                          {
+                            "id": "00112ab3-c5d7-4e56-b555-9497268ff598",
+                            "shortDescription": "Chronic degenerative aortic valve disease (disorder)",
+                            "value": "92808004",
+                            "codeSystem": {
+                              "id": "2f2b4c4c-1e20-4a0d-a3fc-eff4996771e6",
+                              "name": "SNOMED"
+                            }
+                          },
+                          {
+                            "id": "0022432a-b71a-4591-95ff-4ce2415c7ac5",
+                            "shortDescription": "Congenital right ventricular anomaly (disorder)",
+                            "value": "871668002",
+                            "codeSystem": {
+                              "id": "2f2b4c4c-1e20-4a0d-a3fc-eff4996771e6",
+                              "name": "SNOMED"
+                            }
+                          }
+                        ],
+                        "codeSystem": {
+                          "id": "2f2b4c4c-1e20-4a0d-a3fc-eff4996771e6",
+                          "name": "SNOMED"
+                        }
+                      }
+                    ],
+                    "codeListsResolving": [],
+                    "patientAgeConstraints": [
+                      {
+                        "minAgeMonths": 6,
+                        "maxAgeMonths": 0
+                      }
+                    ],
+                    "eventDateConstraint": null,
+                    "eventCountConstraint": null,
+                    "preCoordinatedConstraints": [],
+                    "datasets": [
+                      {
+                        "id": "801b8501-9143-458f-9850-2886a2be69f4",
+                        "name": "General Practice Extraction Service (GPES v4)",
+                        "description": "GPES collects information for a wide range of purposes, including providing GP payments. Versions 1, 4 and 5 have been used to directly identify people and apply specific rules to the cohort.",
+                        "acronym": "GPES v4",
+                        "url": "https://digital.nhs.uk/services/general-practice-extraction-service"
+                      },
+                      {
+                        "id": "4e40980e-27f9-4b78-bdcd-7f9c72b9de74",
+                        "name": "GPES Data for Pandemic Planning and Research COVID-19 (GDPPR)",
```

Comment:
I left that in as it's the name of the dataset, rather than anything that identifies the cohort/condition group. I was working on the basis that the dataset can be attached to it's parent without telling you anything about it.